### DISCUSSION TITLE
[Bug]: Simple Theme navbar height param is not correctly set as variable

### DISCUSSION THREAD

**Discussion by candidexmedia** [GetPublii/Publii#1815] [Bug Report] [answered]
### Operating system

Windows 10

### Publii version

0.46.2 (build 16968)

### Post editor

None

### Bug description

For [Simple theme](https://marketplace.getpublii.com/themes/simple/), version 3.0.1.0

The `theme-variables.js` file, at Line 463 or so, has the navbar height set to ` --navbar-height: 4.4rem;` when it should be set to `--navbar-height: ${params.navbarHeight};`:

![side by side of theme-variables.js and config.json, showing the missing param](https://github.com/user-attachments/assets/315dbc41-597c-40df-8b87-48d40024fdc7)

What happens is that the user sets the navbar height in the Publii GUI, but nothing appears to change in the previewed site.

**Comment by bobmitro** [comment, marked as answer]
All good! The custom value is only applied on larger devices. On mobile, the height is intentionally kept fixed at 4.4rem. 
If something still seems off, let me know! 

<img width="773" alt="screenshot 2024-11-19 at 07 36 30" src="https://github.com/user-attachments/assets/0e8eda44-846c-48c1-87a0-9f988b1d4856">

**Reply by candidexmedia** [reply]
Unfortunately, I didn't see any changes on the desktop version when I adjusted the setting and previewed my site. I did some digging, and changing the `theme-variable.js` fixed it.

This was on Firefox.

**Reply by bobmitro** [reply]
> Unfortunately, I didn't see any changes on the desktop version

Cache? Did you override the theme-variable.js file?
I just checked it a moment ago, and it works perfectly on my end. 

<img width="2562" alt="screenshot 2024-11-19 at 15 59 47" src="https://github.com/user-attachments/assets/c0c3bca8-226e-45ca-840e-0948f09d5e71">

**Reply by candidexmedia** [reply]
Hmm, I'm not sure what happened. I tried it on a fresh website, and it did indeed work... Maybe I made a mistake when editing the  `theme-variable.js` file. Sorry about that!
